src/core_plugins/kibana/public/home/home.less

```diff
@@ -0,0 +1,36 @@
+@import (reference) "~ui/styles/variables.less";
+
+.home {
+  background-color: @globalColorLightestGray;
+  min-height: 100vh;
+}
+
+.homePanel {
+  padding: 24px;
+  background-color: @white;
+}
+
+.topFeatures {
+  margin-top: 12px;
+}
+
+.center {
+  text-align: center;
+}
+
+.featureDirectory {
+  background: @white;
+  margin: 0;
+  border-left: 1px solid @globalColorLightGray;
+  border-right: 1px solid @globalColorLightGray;
+  border-bottom: 1px solid @globalColorLightGray;
+  padding: 24px;
```

Comment:
Make this 16px. You're already getting 8px from the inner cards themselves, which will add up to 24px.

Reply:
Should I update homePanel class as well?

Reply:
Nope. You're padding is fine there.